client/http.go

```diff
@@ -72,3 +72,14 @@ func makeURL(base *url.URL, pathFormat string, args ...interface{}) *url.URL {
 	callURL.Path = path.Join(callURL.Path, fmt.Sprintf(pathFormat, args...))
 	return callURL
 }
+
+// setupURLQuery setups URL query parameters
+func setupURLQuery(u *url.URL, queries map[string]string) *url.URL {
+	q := u.Query()
+	for k, v := range queries {
+		fmt.Println(k, v)
```

Comment:
Please, do not print debug messages.

Reply:
Ah, sry I forgot to remove them.

I will update on this PR.